api/datastore/sql/sql.go

```diff
@@ -915,6 +1035,46 @@ func buildFilterCallQuery(filter *models.CallFilter) (string, []interface{}) {
 	return b.String(), args
 }
 
+func buildFilterFuncQuery(filter *models.FuncFilter) (string, []interface{}) {
+	if filter == nil {
+		return "", nil
+	}
+	var b bytes.Buffer
+	var args []interface{}
+
+	// where(fmt.Sprintf("image LIKE '%s%%'"), filter.Image) // TODO needs escaping, prob we want prefix query to ignore tags
+	args = where(&b, args, "id>?", filter.Cursor)
+
+	fmt.Fprintf(&b, ` ORDER BY id ASC`)
```

Comment:
The swagger specifies results are sorted alphabetically. Though not entirely sure this executes those requests.

Reply:
yea i waffled back and forth on this. can switch it back to that, boy do ids make great cursors though